key-chord.el

```diff
@@ -173,7 +173,7 @@
 ;; ########   History   ########################################
 ;;
 ;; 0.7 (2017-06-29) zk-phi
-;; 	Add option `key-chord-safety-interval'
+;; 	Add option `key-chord-safety-interval-backward/forward'
```

Comment:
You are adding **two** options and neither one is named `key-chord-safety-interval-backward/forward`.

Reply:
I don't make a bump version, so this History entry was the whole disapear.